test/integration/expression-tests/within/basic/test.json

```diff
@@ -0,0 +1,56 @@
+{
+  "expression": ["within", {
+      "type": "Polygon",
+      "coordinates": [[[0, 0], [0, 5], [5, 5], [5, 0], [0, 0]]]
```

Comment:
what about negative tests that would exercise error code path?

Reply:
Added, thanks!